rasa_nlu/evaluate.py

```diff
@@ -726,4 +809,6 @@ def main():
 
 
 if __name__ == '__main__':  # pragma: no cover
+    parser = create_argument_parser()
```

Comment:
This still works as the parser for cmdline_args is instantiated before calling main(). Because you can specify the save-to location of confusion matrix, histogram and error log with the cmdline_args as well, instead of instantiating a parser for cmdline_args in each of the three respective methods and in the main() I thought it would make more sense to do this once in the main block.
Alternatives would be to hardcode the save-to locations and not offer these as arguments or instantiate the args in each method. Pls let me know how you want me to handle this and I will make the changes accordingly.